draft-ietf-tls-sni-encryption.md

```diff
@@ -318,14 +319,14 @@ apply just as well for these transports as for TLS over TCP.
 This points to a requirement for SNI Encryption mechanisms to also
 be applicable to non-TCP transports such as DTLS or QUIC.
 
-## Fail to fronting
+## Mitigate detection by TLS proxies
```

Comment:
Maybe you need to describe the scenario you are concerned about right up front.  That is,

> A TLS proxy is a middlebox that terminates TLS connections from clients for the purposes of monitoring or modifying the content of communications.  This is usually enabled by configuring clients with a trust anchor controlled by the TLS proxy.

Then you can avoid using the gendered Man-in-the-Middle and use "TLS proxy" ;)

Reply:
Removing section 3.9